content/hi/docs/reference/glossary/cgroup.md

```diff
@@ -0,0 +1,16 @@
+---
+title: सीग्रुप (cgroup,control group)
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  वैकल्पिक संसाधन अलगाव, लेखांकन और सीमाओं के साथ Linux प्रक्रियाओं का एक समूह।
+aka:
+tags:
+- fundamental
+---
+वैकल्पिक संसाधन अलगाव, लेखांकन और सीमाओं के साथ Linux प्रक्रियाओं का एक समूह।
+
+<!--more-->  
+
+सीग्रुप एक लिनक्स कर्नेल सुविधा है जो प्रक्रियाओं के संग्रह के लिए रिसोर्स उपयोग (CPU, मेमोरी, डिस्क I/O, नेटवर्क) को सीमित, जिम्मेदार और अलग करती है।
```

Comment:
```suggestion
cgroup एक Linux कर्नल सुविधा है जो प्रक्रियाओं के एक समूह के लिए संसाधन उपयोग (CPU, मेमोरी, डिस्क I/O, नेटवर्क) को अलग, प्रबंधित, और सीमित करता है।
```

Reply:
@KashishLakhara04 Are you still active with this PR? 
@Babapool I think this is not a major issue enough to prevent a merge. I think we can merge this now and make the change with a future PR.

Reply:
Yes, we can create a follow up issue for this PR to fix this. Could the follow up issue be labelled as a good first issue?

Reply:
Yes, a good first issue would be good.